src/Core/src/Handlers/View/ViewHandler.Tizen.cs

```diff
@@ -0,0 +1,67 @@
+﻿using NativeView = ElmSharp.EvasObject;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ViewHandler
+	{
+		static partial void MappingFrame(ViewHandler handler, IView view)
+		{
+			UpdateTransformation(handler, view);
```

Comment:
여기에 Geometry를 설정하는 부분이 빠진듯합니다. (iOS는 UpdateTransformation 안에 포함되어 있네요)
```suggestion
                        ((NativeView?)handler.WrappedNativeView)?.UpdateFrame(view);
			UpdateTransformation(handler, view);
```

Reply:
코드를 더 살펴보니까, Control.Core같은 경우 View의 좌표를 업데이트 하는 부분에서는  `child.Handler?.NativeArrange(child.Frame);` Handler를 NativeArrage를 직접 호출하는 부분이 많네요. 
이게 일부러 그런건지, 단순하게 통일되지 않고 이것저것 사용하는건지 애매하네요. 

즉, View의 좌표를 이동시킬때는 명시적으로 NativeArrage를 사용하게 하고, `Frame`에 대한 업데이트는 Transformation할때만 사용되는건지...

그런데 그것도 잘못된것이, Transformation은 Native 좌표가 업데이트 된 이후에 해야 올바르게 될텐데...

Reply:
결론, Native 좌표를 업데이트 시키는것은 NativeArrage에서만 한다.